### PR TITLE
Fixed black key linear-gradient when clicked

### PR DESCRIPTION
**Pull request by elijaholan**
When the user plays a key through their keyboard, the black keys will also give visual feedback. The following screenshot contains an example for the 'D' key.

image

### CODE REVIEW

**Review by melissalupu — APPROVED** [review]
The linear gradient looks similar to the Figma design!

**Review by Jsimpada — APPROVED** [review]
Looks like the initial design, well done :100:

File: src/components/keyboard/keyboard.js [hunk 1]
```diff
@@ -10,16 +10,23 @@ const useKeyboardBindings = (play, setKey, splash, keys) => {
           if(document.activeElement.id !== 'bpmInput' && keys.includes(e.key)) {
             play({ id: e.key });
             setKey(e.key);
+
+            let element;
+
+            document.getElementById('key-' + e.key) ?
```
Comment (brunogrcsada) [inline src/components/keyboard/keyboard.js:16]: Nice workaround without too much code; clean and simple.

**Review by brunogrcsada — APPROVED** [review]
I was having a few issues with this linear-gradient! Nice solution.

**Pull request merged**